python/ecl/grid/ecl_region.py

```diff
@@ -62,8 +62,12 @@ def select_method(select):
     in i:[0:5] interval:
 
        region = EclRegion( grid , False )
-       region.select_islice(0 , 5)     # Selects all cells with i:[0:5]
-       region.select_jslice(0 , 5)     # Selects all cells with j:[0:5] AND i:[0:5]
+
+       # Select all cells with i:[0:5]:
```

Comment:
What is up with the colons?

Reply:
Probably some sort of preference? How about now? Removed some colons and some whitespace

Reply:
Still don't get what the colons are about, but I guess it is not mission critical anyway.